### ISSUE TITLE
fix MacOS keystore

### ISSUE THREAD

**Issue by kernelkind** [damus-io/notedeck#151]
MacOS keystore impl is behaving strangely. In some cases, when opening the app it is asking for the MacOS user password for every entry in the keystore even after selecting "always allow".

It should only ask for password once per account added when "always allow" is selected, and then never again. If this isn't possible in `security-framework`, a basic file storage impl should be used instead, or possibly try #138 .

will close #137

**Comment by jb55** [comment]
isn't this just a duplicate of #137 ?

**Comment by kernelkind** [comment]
> isn't this just a duplicate of #137 ?

yeah pretty much, I wanted to generalize the issue a bit but I didn't want to hijack it because then your comment would look out of place
